src/azure-cli/azure/cli/command_modules/hdinsight/custom.py

```diff
@@ -490,3 +526,201 @@ def execute_hdi_script_action(cmd, client, resource_group_name, cluster_name,
     ]
 
     return client.execute_script_actions(resource_group_name, cluster_name, persist_on_success, script_actions_params)
+
+
+# pylint: disable=redefined-builtin
+def create_autoscale(cmd, client, resource_group_name, cluster_name, type, min_count=None,
+                     max_count=None, timezone=None, days=None, time=None, count=None):
+    from azure.mgmt.hdinsight.models import Autoscale, AutoscaleCapacity, AutoscaleRecurrence, AutoscaleSchedule, \
+        AutoscaleTimeAndCapacity
+    autoscale_configuration = None
+    if type == 'Load':
+        if not all([min_count, max_count]):
+            raise CLIError(
+                'When the autoscale type is Load, both the min_count and max_count should be specified.')
```

Comment:
>min_count [](start = 59, length = 9)

`min_count` and `max_count` are internal variables, not customer facing parameters. I would suggest to change to `--min-count` `--max--count`

Reply:
changed it to "When the --type is Load, both --min-count and --max-count should be specified."

---
In reply to: [466344819](https://github.com/Azure/azure-cli/pull/14692#discussion_r466344819) [](ancestors = 466344819)